src/gen_dragonfruit/gd_dev_adafruit_MPL115A2_temp.cpp

```diff
@@ -7,10 +7,10 @@ void gd_dev_adafruit_MPL115A2_temp_open(void){
     mpl115a2t1.begin();
 }
 
-int32_t gd_dev_adafruit_MPL115A2_temp_read(void){
-  int32_t value = 0;
+uint16_t gd_dev_adafruit_MPL115A2_temp_read(void){
+  uint16_t value = 0;
   #ifndef SEN_STUB
-  value = mpl115a2t1.getTemperature()*10;
+  value = ((mpl115a2t1.getTemperature() + 273.15) * 100); //Function gives Celcius, return centiKelvin (cK)
```

Comment:
This is a style thing, but it might be good to make it less ambiguous here in regards to types. I would do the following:

* Define a new variable to store the floating point value that `mpl115a2t1.getTemperature()` returns
* Use the uint16_t cast operator to explicitly convert the float to to a uint16_t

This has the advantage of being much less ambiguous for anyone who's looking through the source code in the future.